src/main/java/com/liefery/android/icon_badge/foreground/DrawableForegroundDrawer.java

```diff
@@ -8,19 +8,33 @@
 public class DrawableForegroundDrawer extends ForegroundShapeDrawer {
     private Drawable drawable;
 
+    private int size;
+
+    private float offset;
+
+    private float elevation;
+
     public DrawableForegroundDrawer( @NonNull Drawable drawable ) {
         this.drawable = DrawableCompat.wrap( drawable ).mutate();
     }
 
     @Override
-    public void prepare( int color, int size ) {
+    public void prepare( int color, int size, float elevation ) {
         DrawableCompat.setTint( drawable, color );
-        int offset = (int) Math.round( size * 0.2 );
-        drawable.setBounds( offset, offset, size - offset, size - offset );
+        this.offset = Math.round( size * 0.2 );
```

Comment:
What is this magic number?

Reply:
The icon inset. Basically like a padding.